pid_node:
  ros__parameters:
    kp_yaw: 0.018
    ki_yaw: 0.0
    kd_yaw: 0.0
    kp_pitch: 0.039
    ki_pitch: 0.0
    kd_pitch: 0.0
    kp_roll: 0.018
    ki_roll: 0.0
    kd_roll: 0.0
    kp_x: 0.018
    ki_x: 0.0
    kd_x: 0.0
    kp_y: 0.018
    ki_y: 0.0
    kd_y: 0.0
    kp_z: 0.018
    ki_z: 0.0
    kd_z: 0.0